statefun-e2e-tests/statefun-smoke-e2e-framework/src/main/java/org/apache/flink/statefun/e2e/smoke/Ids.java

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
Will be used by framework hence put in framework module and let java fn depends on the framework code.